javascript/packages/utils/src/misc.ts

```diff
@@ -120,6 +120,12 @@ export function getLokiUrl(
   return loki_url;
 }
 
+export const Timeout = (time: number) => {
```

Comment:
I think we could use a more concrete naming here, something like `TimeoutAbortController`. wdyt?